saga-core/src/main/java/io/servicecomb/saga/core/application/SagaCoordinator.java

```diff
@@ -105,12 +121,14 @@ public void reanimate() {
       Saga saga = new Saga(
           eventStore,
           executorService,
-          recoveryPolicy,
+          recoveryPolicy.get(event.sagaId),
```

Comment:
this will not work, since this method is called on restart

Reply:
Done